desktop/src/main/java/bisq/desktop/primary/main/left/LeftNavView.java

```diff
@@ -298,20 +311,30 @@ private void maybeAnimateMark() {
             double targetY = menuTop + selectedLeftNavButton.getBoundsInParent().getMinY();
             if (selectedLeftNavButton instanceof LeftNavSubButton) {
                 targetY += tradeSubMenuItems.getLayoutY();
+                targetY += walletSubMenuItems.getLayoutY();
```

Comment:
@alvasw I see a few UI issues with wallet submenu:
- when Trade menu is selected Wallet menu is incorrectly highlighted
- when any protocol from Trade submenu is selected then green marker moves too low

Preview:
![navigation-issue](https://user-images.githubusercontent.com/84982606/168023222-fafe6533-e085-4fdb-996f-580dda8a41b4.gif)

Reply:
Oops, I just saw that. You're right. Can you fix it? I have a lot of other things to do at the moment.

Reply:
Yes - I'll be updating this menu soon.

Reply:
Thank you!